common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -597,6 +605,18 @@ static void putObject(Object o, long offset, Object x) {
         UNSAFE.putObject(o, offset, x);
     }
 
+    static void loadFence() {
```

Comment:
I think the Unsafe naming isn't very explanatory. I would either name it 
- loadLoadFence; so you get [LoadLoad]
- loadAcquireFence, so get [LoadLoad][LoadStpre]

Reply:
yeah good point, probably we dont need that anymore, currently we use getIntVolatile and putIntOrdered instead,
I could add this as a comment so it's easier to understand 👍